### DISCUSSION TITLE
Serial connect/disconnect button

### DISCUSSION THREAD

**Discussion by 5ocworkshop** [terjeio/ioSender#124] [Ideas]
First - great application and extremely helpful the way you extract and display the grblHAL config.  Thank you.

It would be helpful to have a serial disconnect/reconnect button on the main IOSender.  When developing and flashing regularly (and still connecting over USB), it would be great to be able to one-click disconnect rather than having to close the app to allow the flash process to proceed, and then relaunch it.

**Comment by terjeio** [comment]
There has been a number of requests to allow connect/disconnect without restarting and I am considering that. The sender configures itself (including the UI) from data requested fom the controller on startup, this has to be changed to allow it at any time. Not easily done but doable.

For boards that allows UART comms on a port that is different from the port the programmer uses the sender can be kept connected at all times for easier testing. I often use a FTDI USB <> UART breakout for that.
